### DISCUSSION TITLE
Roof removal plugin suggestion - Delay before showing a roof that has been removed

### DISCUSSION THREAD

**Discussion by Swepps** [runelite/runelite#17548] [Ideas]
Would it be possible to add a small delay before a roof is shown again after it has been hidden?

I feel like this would be particularly nice when 'Hovered tile' is enabled as moving the mouse over the screen wouldn't flick between the removed and shown state unless the mouse has been off the roof for half a second or so.

**Comment by BigPhysh** [comment]
Was about to make this same suggestion, copied text here: Absolutely love the roof removal plugin, world looks so much better running through cities getting to leave them on and it not be a handicap. However, the roofs flash in and out while moving your mouse around with "hovered tile" on; could you add a custom timer field and checkbox so that after a roof is hidden, it doesn't reappear for x seconds, with the timer refreshing every time you would hide the roof? This way if I hide a roof by walking in, hovering over, etc, it might stay hidden for 5-10seconds or so before it comes back.

**Comment by BanJoeH** [comment]
This would be such a nice quality of life. A configurable delay for each hide (probably very small, but nice for when quickly moving the mouse over a building) and show (probably a longer delay to reduce the flashes) would feel so nice
